content/web-api-rest/exercises/_index.md

```diff
@@ -0,0 +1,215 @@
+---
+title: "Exercises: Web APIs and REST"
+date: 2022-12-15T09:16:07-06:00
+draft: false
+weight: 4
+originalAuthor: John Woolbright # to be set by page creator
+originalAuthorGitHub: jwoolbright23 # to be set by page creator
+reviewer: Kimberly Horan # to be set by the page reviewer
+reviewerGitHub: codinglikeagirl42 # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: 12/15/22 # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+We've created a CodingEvents API application [here](https://github.com/LaunchCodeEducation/coding-events-api/tree/1-sqlite). Before we ask you to fork and clone this, let's first think conceptually about the kinds of requests a client application could make to this API.
+
+## The API
+
+This branch of `coding-events-api` starts by only exposing a single resource, `CodingEvent`, and four endpoints for interacting with it.
```

Comment:
```suggestion
The 1-sqlite branch of `coding-events-api` starts by only exposing a single resource, `CodingEvent`, and four endpoints for interacting with it.
```

Reply:
Resolved this by removing the 1-sqlite branch altogether and having the learners work off of main.